### ISSUE TITLE
rec_r34_vd_tps_bilstm_attn(from PaddleOCR repo) MKLDNN prediction failed

### ISSUE THREAD

**Issue by OliverLPH** [PaddlePaddle/Paddle#27398]
System information
-PaddlePaddle version: develop, `d28162b97fd2d224968c18c9da735900ef280e7c`
-CPU: Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz, mkldnn enabled
-GPU: P4, CUDA10, CUDNN7.5, compile with GPU, but run cpu inference
-OS Platform: Ubuntu1604
-Dockerfile: Paddle/tools/manylinux1/Dockerfile.cuda10_cudnn7_gcc48_ubuntu16
-Python version: 3.7
-Cmake orders
-C++version.txt
-API information

```python
config = AnalysisConfig(args.model_file, args.params_file)
config.disable_gpu()
config.switch_use_feed_fetch_ops(False)
config.switch_specify_input_names(True)
config.enable_mkldnn()
```
Note: You can get most of the information by running summary_env.py.

Steps to reproduce the behavior
[mkldnn_reproduce_code.txt](https://github.com/PaddlePaddle/Paddle/files/5244239/mkldnn_reproduce_code.txt)

```shell
python3.7 mkldnn_reproduce_code.txt --model_file=./rec_r34_vd_tps_bilstm_attn/__model__ \
                                    --params_file=./rec_r34_vd_tps_bilstm_attn/__params__
```

Describe your current behavior

```shell
I0918 07:50:21.910486 27984 analysis_predictor.cc:527] ======= optimize end =======
I0918 07:50:21.910612 27984 naive_executor.cc:102] ---  skip [feed], feed -> image
I0918 07:50:21.911798 27984 naive_executor.cc:102] ---  skip [save_infer_model/scale_0.tmp_0], fetch -> fetch
I0918 07:50:21.911818 27984 naive_executor.cc:102] ---  skip [save_infer_model/scale_1.tmp_0], fetch -> fetch
W0918 07:50:21.927631 27984 operator.cc:205] pool2d raises an exception dnnl::error, could not create a descriptor for a pooling forward propagation primitive
Traceback (most recent call last):
  File "test_zerotensor.py", line 44, in <module>
    main()
  File "test_zerotensor.py", line 20, in main
    predictor.zero_copy_run()
RuntimeError: could not create a descriptor for a pooling forward propagation primitive
```

Code to reproduce the issue
[mkldnn_reproduce_code.txt](https://github.com/PaddlePaddle/Paddle/files/5244239/mkldnn_reproduce_code.txt)

Other info / logs
![image](https://user-images.githubusercontent.com/14963836/93573396-a7f7d080-f9c9-11ea-8725-a5fee08affb7.png)

**Comment by luotao1** [comment]
@lidanqing-intel @jczaja Please help see this OCR issue

**Comment by lidanqing-vv** [comment]
@luotao1  Where is the model file located ?

**Comment by lidanqing-vv** [comment]
model link : https://sys-p0.bj.bcebos.com/inference/python-ocr-infer.tgz

**Comment by jczaja** [comment]
@luotao1 , @lidanqing-intel I reproducd the problem. Root cause is that oneDNN pool2d kernel is not supporting "adpative" pooling so this is something that need to be added to fix this issue

**Comment by jczaja** [comment]
@luotao1 This issue to be fixed require two PRs to be merged: 1) #27770 (merged) 2) #27747 (awaiting approval)
